tests/Unit/Contrib/IdConverterTest.php

```diff
@@ -12,6 +12,14 @@
  */
 class IdConverterTest extends TestCase
 {
+    //Based on this section of the Jaeger spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/jaeger.md#ids
+    public function test_correctly_converts_example_from_spec()
+    {
+        $hex = 'FF00000000000000';
+
+        $this->assertEquals(-72057594037927936, IdConverter::convertOtelToJaegerSpanId($hex));
```

Comment:
do any of the other implementations have a set of well-known or edge inputs and their correct conversion? This feels like a good place to implement a phpunit `@dataProvider`

Reply:
I added the 1 (non-edge-case) test case from Go in. Let me take a look at some of the others to see if they have edge cases already.

Reply:
Didn't see any tests for this in Java or Python (maybe I wasn't searching in the right spots though). Found 1 case in JS but it's not an edge case - https://github.com/open-telemetry/opentelemetry-js/blob/a824578c981e02a762561db0b6bb44d24af1229d/packages/opentelemetry-exporter-jaeger/test/transform.test.ts#L180